### PR TITLE
Loads `p5.min.js` for `CodeFrame` iframes using manually created script

### PR DESCRIPTION
**Pull request by howard-e**
Address #213. 

This PR updates how `CodeFrame` created `iframe`s make use of `p5.min.js`.

It avoids requiring a _potentially_ cached version of the `p5.min.js` script, by instead passing the text content of the script directly to the iframe to be included in a newly created script element.

### CODE REVIEW

**Comment by stalgiag** [comment]
@howard-e I think that the build is failing because `client:load` is now necessary on the `CodeFrame` inside the `SketchEmbed` component. This is untested.

**Comment by howard-e** [comment]
@stalgiag https://github.com/bocoup/p5.js-website/pull/224/commits/4b4be8db294c21abe087f2d5497ffb23239dd2ea should address that build error.

It also reflects our earlier discussion on handling a future optimization effort where `client:only` will be need to be included on `<SketchEmbed>`.

File: src/components/CodeEmbed/frameForServer.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+export const CodeFrameForServer = (props: CodeFrameProps) => {
+  const iframeRef = useRef<HTMLIFrameElement>(null);
+  const containerRef = useRef<HTMLDivElement>(null);
+
+  // For performance, set the iframe to display:none when
+  // not visible on the page. This will stop the browser
+  // from running `draw` every frame, which helps performance
+  // on pages with multiple sketches, and causes sketch
+  // animations only to start when they become visible.
+  useLayoutEffect(() => {
```
Comment (stalgiag) [inline src/components/CodeEmbed/frameForServer.tsx:61]: Nice addition!

**Review by stalgiag — APPROVED** [review]
Works well! Thank you!

**Pull request merged**